docs/content/docs/get-started/quickstart/react_agent.md

```diff
@@ -22,6 +22,155 @@ specific language governing permissions and limitations
+
+- **Review Analysis**: Processes a stream of product reviews and uses a single agent to extract a rating (1–5) and unsatisfied reasons from each review. And notify the shipping manager if a review includes dissatisfaction with the shipping process.   
+
+
+## Example Explain
+
+### Prepare Agents Execution Environment
+
+Create the agents execution environment, and resister the available chat model connections and tools to the environment. 
+```python
+# Set up the Flink streaming environment and the Agents execution environment.
+env = StreamExecutionEnvironment.get_execution_environment()
+agents_env = AgentsExecutionEnvironment.get_execution_environment(env)
+
+# Add Ollama chat model connection and notify shipping manager tool to be used
+# by the Agent.
+agents_env.add_resource(
+    "ollama_server",
+    ResourceDescriptor(clazz=OllamaChatModelConnection, request_timeout=120),
+).add_resource(
+    "notify_shipping_manager", Tool.from_callable(notify_shipping_manager)
+)
+```
+
+### Create the ReAct Agent
+Create the ReAct Agent instance, configure the chat model, prompt and the output schema of result to be used.
+```python
+review_analysis_react_agent = ReActAgent(
+    chat_model=ResourceDescriptor(
+        clazz=OllamaChatModelSetup,
+        connection="ollama_server",
+        model="qwen3:8b",
+        tools=["notify_shipping_manager"],
+    ),
+    prompt=review_analysis_react_prompt,
+    output_schema=ProductReviewAnalysisRes,
+)
+```
+
+### Integrate the ReAct Agent with Flink
+Produce a source DataStream by reading a product review text file, and use the ReAct Agent to analyze the review and generate result DataStream. Finally print the result DataStream.
+```python
+# Read product reviews from a text file as a streaming source.
+# Each line in the file should be a JSON string representing a ProductReview.
+product_review_stream = env.from_source(
+    source=FileSource.for_record_stream_format(
+        StreamFormat.text_line_format(),
+        f"file:///{current_dir}/resources/",
+    )
+    .monitor_continuously(Duration.of_minutes(1))
+    .build(),
+    watermark_strategy=WatermarkStrategy.no_watermarks(),
+    source_name="streaming_agent_example",
+).map(
+    lambda x: ProductReview.model_validate_json(
+        x
+    )  # Deserialize JSON to ProductReview.
+)
+
+# Use the ReAct agent to analyze each product review and notify the shipping manager
+# when needed.
+review_analysis_res_stream = (
+    agents_env.from_datastream(
+        input=product_review_stream, key_selector=lambda x: x.id
+    )
+    .apply(review_analysis_react_agent)
+    .to_datastream()
+)
+
+# Print the analysis results to stdout.
+review_analysis_res_stream.print()
+```
+
+## Prerequisites
+
+* Unix-like environment (we use Linux, Mac OS X, Cygwin, WSL)
+* Git
+* Java 11
+* Python 3.10 or 3.11
+
+## Preparation
+
+### Prepare Flink
+
+Download a stable release of Flink 1.20.3, then extract the archive:
```

Comment:
This remind me that, shall we also mention in the Installation page, that we require Flink version 1.20.3?